packages/react/src/components/DashboardEditor/DashboardEditorCardRenderer.jsx

```diff
@@ -288,9 +168,6 @@ const DashboardEditorCardRenderer = React.memo(
       // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [cardProps.values?.hotspots?.length, cardProps.onFetchDynamicDemoHotspots]);
 
-    if (!isCardJsonValid(cardProps)) {
-      return renderDefaultCard(cardProps);
-    }
```

Comment:
This is being moved into the new `DashboardEditorDefaultCardRender` component below: https://github.com/carbon-design-system/carbon-addons-iot-react/pull/2790/files#diff-1cb56abaf3f9f980a20b247c77f76852c3d99620198c40d494aed49883c3924fR45